packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -789,6 +789,11 @@ export default class TradeStore extends BaseStore {
         }
     }
 
+    @computed
+    get is_synthetics_available() {
+        return !!this.active_symbols?.find(item => item.market === 'synthetic_index');
+    }
+
```

Comment:
Please remove `is_synthetics_unavailable` in client-store.js as we do not use it anymore and adopt the code accordingly. 🙏🏼

Reply:
@matin-binary  just updated it.